docs/source/export.md

```diff
@@ -137,7 +140,7 @@ formats are supported.
   ```python
   import torch
 
-  model = torch.load("my_exported_model.pt")
+  model = torch.load("my_exported_model.pt", weights_only=True)
```

Comment:
```suggestion
  model = torch.load("my_exported_model.pt")
```

I think `weights_only=True` should only be used when loading a state dict. In this case here we are actually loading a full model class which will fail with `weights_only=True`.